isotovideo

```diff
@@ -73,19 +74,24 @@ bmwqemu::clean_control_files();
 my $init = 1;
 
 my $cpid;
+my $backend;
 
 # all so ugly ...
 sub signalhandler {
 
-    # do not start a race about the results between the threads
-
     my $sig = shift;
-    diag("signalhandler $$: got $sig");
+    print STDERR "$$: signalhandler got $sig\n";
```

Comment:
that's what I am saying, diag is writing to autoinst-log.txt but "print STDERR" might print to syslog so could do
```perl
sub super_important_log {
    my (@args) = @_;
    diag(@args);
    print STDERR "@args\n";
}
…
super_important_log("signalhandler $$: got $sig");
```